library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -68,15 +86,16 @@ sealed class Conversation {
             return when (this) {
                 is V1 -> null
                 is V2 -> conversationV2.keyMaterial
+                is Group -> null
             }
         }
 
     fun consentState(): ConsentState {
-        val client: Client = when (this) {
-            is V1 -> conversationV1.client
-            is V2 -> conversationV2.client
+        return when (this) {
+            is V1 -> conversationV1.client.contacts.consentList.state(address = peerAddress)
+            is V2 -> conversationV2.client.contacts.consentList.state(address = peerAddress)
+            is Group -> ConsentState.UNKNOWN // No such thing as consent for a group
```

Comment:
Or should this always have the consent state as Allowed for now?

Reply:
Assuming that implementors are not going to persist it, it makes sense to hardcode it to ALLOWED, so that it shows up correctly in their UI, and then use the true value once we have implemented consent

Reply:
I actually think its probably safer to keep this unknown for now since when they move from request to inbox it would feel like a broken experience once it changes to the network based version. Probably should prioritize group chat consent sooner. 🤔

Reply:
Group chat consent doesn't seem that bad. I'm tempted to sneak it in before Denver (on top of our existing consent mechanism, just with some new values)